src/lib/zf/zf_stackdump.c

```diff
@@ -26,6 +26,41 @@ static void dump_zockets(SkewPointer<zf_stack> stack, Zocket* zockets,
   }
 }
 
+void dump_attributes(SkewPointer<zf_stack_impl> stimpl)
+{
+  zf_dump("---------------------attributes-------------------------------\n");
+  if(stimpl->sti_tx_ring_max==-1){
+  zf_dump("tx_ring size= 512\n");
+  }
+  else{
+    zf_dump("tx_ring size= %d\n", stimpl->sti_tx_ring_max);
+  }
+  if(stimpl->sti_rx_ring_max==-1){
+  zf_dump("rx_ring size= 512\n");
+  }
+  else{
+    zf_dump("rx_ring size= %d\n", stimpl->sti_rx_ring_max);
+  }
+  zf_dump("tx_timestamping= %d\n", stimpl->sti_tx_timestamping);  
```

Comment:
nit: it seems odd that we would show "attribute= value" here, where other values seem to be printed without this extra space after the equals sign ("attribute=value"). For example, from your provided dump we see things like `pkt_bufs_n=21632` rather than `pkt_bufs_n= 21632`:
```
============================================================
onload version=27eb4307c8 2023-11-15 onload-8.1.2
tcpdirect version=8.1.3 onload-8.1 tcpdirect-8.1 0c08e35 2024-05-03 v8_1
name=ens1f0/10a interface=ens1f0 vlan_id=65535
  pool: pkt_bufs_n=21632 free=17537
```
I think zf_stackdump is consumed by some things to gather data automatically, so it would probably be good to keep this consistent with the other bits.